EDU-2367-EN-Create-API-overview (#356)
aziontech/docs: src/content/docs/en/pages/main-menu/additional-resources/developer-tools/getting-started-api.mdx

---
title: Get started with Azion API
description: Learn more about official endpoints, methods, and API use cases of the Azion RESTful API.
meta_tags: azion, rest api, restful api, azion restful api
namespace: documentation_products_azion_api_overview
permalink: /documentation/products/azion-api-overview/
---

import Button from '~/components/Button.astro'

This documentation will guide you on the first steps of using the **Azion API**.

## API overview

The **Azion API** is a *RESTful API* based on *HTTPS* requests, that allows users to fully interact with Azion products through API requests. The complete Azion API collection is on Postman.

<Button href="https://api.azion.com/" text="Go to Azion API documentation"></Button>

> Most API endpoints have the main HTTP methods (`GET`, `POST`, `PUT`, `PATCH`, and `DELETE`) so you can create, read, update, and delete Azion configuration instances. All changes made through the API are reflected on the [Real-Time Manager (RTM)](https://manager.azion.com).

Reviewer: ```suggestion
Most API endpoints have the main HTTP methods (`GET`, `POST`, `PUT`, `PATCH`, and `DELETE`) so you can create, read, update, and delete Azion configuration instances. All changes made through the API are reflected on [Real-Time Manager (RTM)](https://manager.azion.com).
```

Reviewer: there's no _the_ before RTM.

:::tip
Access the [API introduction](https://api.azion.com/#intro) documentation to find reference information about the API.
:::

## Prerequisites

To run **Azion API** requests you need:

> - [Real-Time Manager (RTM)](/en/documentation/products/accounts/creating-account/) account.

Reviewer: ```suggestion
- A [Real-Time Manager (RTM)](/en/documentation/products/accounts/creating-account/) account.
```

- A [personal token](/en/documentation/products/accounts/personal-tokens/).

## Make API requests
 

> Azion API documentation offers code examples to make requests with the terminal, using the curl command, or to automate them with programming languages. 

Reviewer 2: ```suggestion
The [Azion API introduction](https://api.azion.com/) offers code examples to make requests with the terminal, using the cURL command, or to automate them with programming languages. 
```

Postman offers HTTPS request snippets for many programming languages.

> Navigate to the `Example Request` to change from curl to another desired programming language.

Reviewer 2: ```suggestion
Navigate to the `Example Request` to change from curl to another desired programming language.

You may also clone the collection and run it in Postman by clicking the **Run in Postman** button.
```
